Add document for aws endpoint setting (#965)
kedacore/keda-docs: content/docs/2.9/scalers/aws-dynamodb-streams.md

**Parameter list:**

- `awsRegion` - AWS Region for the DynamoDB.

> - `awsEndpoint` - An optional endpoint URL that overrides the default AWS endpoint.

Reviewer: ```suggestion
- `awsEndpoint` - Endpoint URL to override the default AWS endpoint. (Default: `""`, Optional)
```
